### PR TITLE
Added ability to set user for postgres

### PR DESCRIPTION
**Pull request by bestDvlpr**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,7 @@
 registry=exadel/
-postgres_password=postgres
+postgres_user=new_postgres
+postgres_password=new_postgresql
+postgres_db=frs_new
```
Comment (pospielov) [inline .env:4]: Why it has other names?

File: dev/init_db.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+su -u postgres
```
Comment (pospielov) [inline dev/init_db.sh:2]: At what point and who runs this script? 
I thought postgres docker container should do it by itself

File: .env (hunk 1 above)
Comment (bestDvlpr) [reply .env]: I have done corrections of them

File: dev/init_db.sh (hunk 2 above)
Comment (bestDvlpr) [reply dev/init_db.sh]: I have re-read about it and actually you are right. I have removed the script.

File: .env [hunk 3]
```diff
@@ -1,5 +1,7 @@
 registry=exadel/
+postgres_user=postgres
```
Comment (pospielov) [inline .env:2]: In this file the name is `postgres_user` and in `dev/.env` it's `postgres_username`

Comment (bestDvlpr) [reply .env]: I have corrected them.

**Review by pospielov — APPROVED** [review]

**Pull request merged**